packages/client/src/config/index.ts

```diff
@@ -1,35 +1,50 @@
 import Topic from "../models/topic";
 import { TopicType } from "../types";
 
+/**
+ * PreQuestions enable us to configure custom questions before the IMTR questions. We will use these questions to customise the Outcome Section.
+ *
+ * Direct importing and including components does not work because the hooks lose context.
+ */
+export enum PreQuestionComponent {
```

Comment:
Changes in this file are great 👍 
Not sure if I would place this enum here. Where do you think it fits best?

Reply:
I agree with you it's not very logical. I think a new file would make it overly complex. What about placing this in `client/src/types.ts` ? I will merge this now and add it to the list, ok?

Reply:
Fixed in #857